app/controllers/ideas_controller.rb

```diff
@@ -20,7 +20,17 @@ def index
 
   # GET /ideas/1
   # GET /ideas/1.json
-  def show; end
+  def show
+    if current_user.nil? && @idea.basket != "New"
```

Comment:
Should also check for rejected ideas so that they can't be searched by iterating id:s